src/components/userDasboard/container/UserDashboard.jsx

```diff
@@ -43,7 +43,7 @@ export class UserDashboard extends Component {
 
   gotoDictionary = (url) => {
     this.props.history.push(url);
-  }
```

Comment:
@muhozi, Is this a formatting change?

Reply:
@dondrzzy Thanks for pointing this out. This is not related to the ticket. I reverted this change.